games/minetest-devel/Portfile

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.1
+
+compiler.cxx_standard   2011
+compiler.thread_local_storage   yes
+
+name                    minetest-devel
+github.setup            minetest minetest 5.6.1
+revision                0
+
+# the game version could be different to the minetest version
+set game_version        5.6.1
+
+# to add on more files to a github portgroup download
+# have to cache the preset distfiles from the github PG
+set main_distfile       ${distfiles}
+
+# then add another distfile, with a direct URL as can't use the github PG again
+set game_distfile       ${game_version}${extract.suffix}
+set game_mastersite     https://github.com/minetest/minetest_game/archive/
+
+distfiles               ${main_distfile}:main \
+                        ${game_distfile}:game
+
+master_sites            ${github.master_sites}:main \
+                        ${game_mastersite}:game
+
+checksums               ${main_distfile} \
+                        rmd160  7b32331e0007f29003107592522d1c906935e44d \
+                        sha256  425d8e9363212dcb5bc7986dd12460c9c3784aca22eb1b90dad6a7fc2c6ef546 \
+                        size    9925064 \
+                        ${game_distfile} \
+                        rmd160  73d35423b856d141cebfa16396fe5dcf834dcf89 \
+                        sha256  5dc857003d24bb489f126865fcd6bf0d9c0cb146ca4c1c733570699d15abd0e3 \
+                        size    2590582
+
+# rename directory - from github portgroup
+post-extract {
+    if {[file exists [glob -nocomplain ${workpath}/${github.author}-${github.project}-*]] && \
+        [file isdirectory [glob -nocomplain ${workpath}/${github.author}-${github.project}-*]]} {
+        move [glob ${workpath}/${github.author}-${github.project}-*] ${workpath}/${distname}
+    }
+}
+
+license                 EUPL-1.2
+categories              games
+platforms               darwin
+maintainers             {macports.org:Zweihorn @Zweihorn}
```

Comment:
You have made bo previous contribution to MacPorts as far as I can tell. So becoming a MacPorts member at this time has zero change, so please provide your obfuscated e-mail address (if you want you can make a dedicated e-mail for this purpose with Gmail or another provider).

Reply:
THX for the advice and I might have to recall the sins of my past obviously.

However, I first would like to clarify the situation and like to avoid any misunderstanding.
1. If I understand you correctly, there is the claim of commits from the past which show my personal email address already?
2. An indication of the maintainer just by a sole GitHub account like `@Zweihorn` would be insufficient?
3. An obfuscated email like `macports.org:Zweihorn` could point at my existing personal email and would greatly ease my life as it allows filtering by the "TO:" address obviously. Or am I wrong?

**So please reconsider** my request for a macports.org email or would there be some obstacles I might be innocently unaware of?

My sincere thanks in advance and I hope to not bother you.

Reply:
For the moment I stick to just `@Zweihorn` for the benefit of the example and will await your review.